services/audit-service/src/controllers/export-logs.controller.ts

```diff
@@ -0,0 +1,49 @@
+import {service} from '@loopback/core';
+import {Filter, repository} from '@loopback/repository';
+import {param, get, getModelSchemaRef} from '@loopback/rest';
+import {
+  OPERATION_SECURITY_SPEC,
+  STATUS_CODE,
+  CONTENT_TYPE,
+} from '@sourceloop/core';
+import {authenticate, STRATEGY} from 'loopback4-authentication';
+import {authorize} from 'loopback4-authorization';
+import {PermissionKey} from '../enums/permission-key.enum';
+import {AuditLog} from '../models';
+import {AuditLogRepository} from '../repositories';
+import {AuditLogExportService} from '../services';
+
+const basePath = '/exportLogs';
+
+export class ExportLogsController {
+  constructor(
+    @repository(AuditLogRepository)
+    public auditLogRepository: AuditLogRepository,
+    @service(AuditLogExportService)
+    public auditLogExportService: AuditLogExportService,
+  ) {}
+
+  @authorize({
+    permissions: [PermissionKey.ExportLogs, PermissionKey.ExportLogsNum],
+  })
+  @authenticate(STRATEGY.BEARER)
+  @get(`${basePath}`, {
```

Comment:
Why not use it as is?
```suggestion
  @get(basePath, {
```